docs_zh-CN/tutorials/customize_mot_model.md

```diff
@@ -1 +1,276 @@
 ## 自定义多目标跟踪模型
+
+我们通常将模型组件分为5类：
+
+- 跟踪器：利用以下组件线索来联系视频帧间目标的组件。
+- 检测器：通常是从一张图片中检出物体的检测组件，例如：Faster R-CNN。
+- 运动估计器：计算相邻帧运动信息的组件，例如：卡尔曼滤波器。
+- 重识别器：从裁剪图片中抽取特征的的独立重识别模型，例如：BaseReID。
+- 跟踪头：用于抽取跟踪线索但是和检测器共享骨干网络的组件。例如：一个特征分支头或者回归分支头。
+
+### 增加一个新的跟踪器
+
+#### 1. 定义一个跟踪器
+
+创建一个新文件 `mmtrack/models/mot/trackers/my_tracker.py`。`BaseTracker` 是提供跨视频跟踪基础 API 的基类，详情请参考 [BaseTracker](https://github.com/open-mmlab/mmtracking/tree/master/mmtrack/models/mot/trackers/base_tracker.py)。 我们推荐新的跟踪器继承该类。
+
+```python
+from mmtrack.models import TRACKERS
+from .base_tracker import BaseTracker
+
+@TRACKERS.register_module()
+class MyTracker(BaseTracker):
+
+    def __init__(self,
+                 arg1,
+                 arg2,
+                 *args,
+                 **kwargs):
+        super().__init__(*args, **kwargs)
+        pass
+
+    def track(self, inputs):
+        # implementation is ignored
+        pass
+```
+
+#### 2. 引入模块
+
+你可以在 `mmtrack/models/mot/trackers/__init__.py` 中加入下面一行。
+
+```python
+from .my_tracker import MyTracker
+```
+
+或者，为了避免更改原始代码，你还可以在 config 文件中增加以下几行来实现：
+
+```python
+custom_imports = dict(
+    imports=['mmtrack.models.mot.trackers.my_tracker.py'],
+    allow_failed_imports=False)
+```
+
+#### 3. 更改原始 config文件
+
+```python
+tracker=dict(
+    type='MyTracker',
+    arg1=xxx,
+    arg2=xxx)
+```
+
+### 增加一个新的检测器
+
+请参考[MMDetection教程](https://mmdetection.readthedocs.io/en/latest/tutorials/customize_models.html)来开发新检测器
+
+### 增加一个新的运动估计器
+
+#### 1. 定义一个运动估计模型（例如：MyFlowNet）
+
+新建一个文件 `mmtrack/models/motion/my_flownet.py`。
+
```

Comment:
forgot to translate `You can inherit the motion model from BaseModule in mmcv.runner if it is a deep learning module, and from object if not.`